newIDE/app/public/index.html

```diff
@@ -50,6 +50,15 @@
           opacity: 1;
         }
       }
+
+      @keyframes verticalEaseShake {
```

Comment:
Put it in newIDE/app/src/UI/Theme/Global/Animation.css

(the text-focus-in is for the message which is shown super early in the loading steps)